### DISCUSSION TITLE
How do i use Glassmorphism .

### DISCUSSION THREAD

**Discussion by dipanshuhappy** [inkline/inkline#305] [Need help]
I saw Glassmorphism in the main website .. i would like to know how will i do it

**Comment by alexgrozav** [comment]
Hey! The glassmorphism example is a using a set of variables and overrides that can be found here: https://github.com/inkline/inkline.io/blob/main/src/components/sections/index/IndexDesignSystemSection.vue

Unfortunately, it's not a complete theme but mostly an example of what you can do with Inkline.
